addons/website/views/website_templates.xml

```diff
@@ -223,6 +223,24 @@
     </ul>
 </template>
 
+<!-- Menu Items -->
+<template id="menu_items" name="Menu Items">
+    <!-- AutoHideMenu: Initially, we need to show menu items forced on one line (with hidden overflow).
+    This will cause some ugly flickering with many menu items (since the "#top_menu" contains items
+    that don't need to be folded too), so we need to add "#top_menu_items" to hide the overflow before
+    unfoldable elements... ("#top_menu_items" will be removed onStart). -->
+    <li id="top_menu_items">
+        <ul>
+            <t t-foreach="submenu_items" t-as="submenu">
```

Comment:
I don't understand, can you show me what you mean? In any case, we really should not alter the website menu structure just for the sake of the autohide menu feature. Especially since we are reworking all those templates and we don't want all the conflicts (but of course, if required we will consider it). If you can explain why this is impossible without changing the structure, we can delay this task and have your work on something else meanwhile.

Reply:
It's simply because we can't hide overflow before unfoldable items other way > without the `#top_menu_items` we have this result onLoad:

![master-menu-overflow-mou](https://user-images.githubusercontent.com/25032876/130049034-a48a6703-d6fe-4104-838a-8f5f3af04bbd.png)

So we need to add a container to have the same structure as after autohideMenu in JS:

![master-menu-structure-mou](https://user-images.githubusercontent.com/25032876/130048932-4c50deb7-5f33-42b2-8a3c-b028fc5c120a.png)